core/src/test/scala/unit/kafka/server/DynamicBrokerConfigTest.scala

```diff
@@ -609,7 +609,10 @@ class DynamicBrokerConfigTest {
     val zkClient: KafkaZkClient = mock(classOf[KafkaZkClient])
     when(zkClient.getEntityConfigs(anyString(), anyString())).thenReturn(new java.util.Properties())
 
-    val oldConfig =  KafkaConfig.fromProps(TestUtils.createBrokerConfig(0, TestUtils.MockZkConnect, port = 9092))
+    val initialProps = TestUtils.createBrokerConfig(0, TestUtils.MockZkConnect, port = 9092)
```

Comment:
Can you explain this change?

Reply:
The `TestDynamicThreadPool` mock class expects the various thread count props to be initialized to the defaults. So I am removing the static overrides.